libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/location/LoggingLocation.java

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.services.android.navigation.v5.location;
+
+public class LoggingLocation {
```

Comment:
Could we rename this DTO? IMO `LoggingLocation` is not reflecting well its purpose.

Reply:
BTW, do we want to expose this? I don't 👀 any use cases other than logging.